src/compressor.py

```diff
@@ -76,6 +77,12 @@ def _compress_images(self):
         GLib.idle_add(self.c_enable_compression, True)
 
     def run_command(self, command, result_item):
+        if not self.do_new_file:
+            # Creates a copy of the input file
+            # This is done in case the output file is larger than the input file
+            temp_filename = result_item.filename + ".temp"
```

Comment:
Maybe a '.' in front of the filename could be useful to "hide" the file in file managers?

Reply:
Good idea. I added this.